zppy/__main__.py

```diff
@@ -59,26 +59,26 @@ def main():
         if tmp.startswith("compy"):
             machine = "compy"
             environment_commands = (
-                "source /share/apps/E3SM/conda_envs/load_{}_e3sm_unified.sh".format(
+                "source /share/apps/E3SM/conda_envs/load_{}_e3sm_unified_compy.sh".format(
                     config["default"]["e3sm_unified"]
                 )
             )
         elif tmp.startswith("cori"):
             machine = "cori"
-            environment_commands = "source /global/cfs/cdirs/e3sm/software/anaconda_envs/load_{}_e3sm_unified.sh".format(
+            environment_commands = "source /global/cfs/cdirs/e3sm/software/anaconda_envs/load_{}_e3sm_unified_cori.sh".format(
```

Comment:
This one needs to be `source /global/common/software/e3sm/anaconda_envs/load_{}_e3sm_unified_cori-haswell.sh`.

Reply:
Good catch, @forsyth2 . The cori path was fixed in the new commit.